apps/editor/__tests__/specs/i18n.test.tsx

```diff
@@ -24,8 +24,10 @@ describe('i18n', () => {
   })
 
   test('should init i18next plugins', () => {
+    const initSpy = jest.spyOn(i18n, 'init').mockImplementation((): any => {
```

Comment:
i18n started to modify values inside the init function somehow which caused the tests to fail. Overriding the implementation means that `.toHaveBeenCalledWith` is correctly working